docs/pages/example/visualize-population-density.html

```diff
@@ -0,0 +1,53 @@
+<div id='map'></div>
+<script>
+var map = new mapboxgl.Map({
+    container: 'map', // container id
+    style: 'mapbox://styles/mapbox/streets-v9', // stylesheet location
+    center: [30.0222, -1.9596], // starting position [lng, lat]
+    zoom: 7 // starting zoom
+});
+
+map.on('load', function () {
+
+    map.addLayer({
+        'id': 'rwanda-provinces',
+        'type': 'fill',
+        'source': {
+            'type': 'geojson',
+            'data': 'https://www.mapbox.com/mapbox-gl-js/assets/rwanda-provinces.geojson'
+        },
+        'layout': {},
+        'paint': {
+            'fill-color':
+            ["let", "density", ["/", ["get", "population"], ["get", "sq-km"]],
+                [
+                    "interpolate",
+                    ["linear"],
+                    ["zoom"],
+                    8,
+                    [
+                        "interpolate",
+                        ["linear"],
+                        ["var", "density"],
+                        274,
+                        ["to-color", "#edf8e9"],
+                        1551,
+                        ["to-color", "#006d2c"]
+                    ],
+                    10,
+                    [
+                        "interpolate",
+                        ["linear"],
+                        ["var", "density"],
+                        274,
+                        ["to-color", "#eff3ff"],
```

Comment:
I *think* @ChrisLoer 's recent work makes the explicit cast to a color here unnecessary

Reply:
I think the cast is necessary until https://github.com/mapbox/mapbox-gl-js/pull/7301 has merged and is being used by the example page, right?

Reply:
LOL, oh yeah. I was working on that. Fridays.